marketplace-ui/src/assets/i18n/de.yaml

```diff
@@ -26,7 +26,18 @@ common:
   error:
     code: 'ERROR CODE'
     oops: 'Oops!'
-    description: 'Unsere Webseite macht eine kleine Pause. Offenbar finden wir die von Dir angefragte Seite nicht.'
+    fullMessage: "{{ errorMessage }} Du tust uns einen großen Gefallen, wenn Du es hier meldest: <a href='mailto:{email}'>support@axonivy.com</a>"
+    description:
+      400: 'Bad Request: Der Server konnte die Anfrage aufgrund fehlerhafter Syntax nicht verstehen..'
+      401: 'Unauthorized: Eine Authentifizierung ist erforderlich und ist fehlgeschlagen oder wurde nicht bereitgestellt.'
+      403: 'Forbidden: Der Server hat die Anfrage verstanden, verweigert jedoch die Autorisierung.'
+      404: 'Not Found: Unsere Webseite macht eine kleine Pause. Offenbar finden wir die von Dir angefragte Seite nicht.'
+      408: 'Request Timeout: Der Server hat das Zeitlimit für die Anfrage überschritten.'
+      500: 'Internal Server Error: Der Server stieß auf ein unerwartetes Problem.'
+      502: 'Bad Gateway: Der Server hat eine ungültige Antwort von einem vorgelagerten Server erhalten.'
+      503: 'Service Unavailable: Der Server kann vorübergehend  die Anfrage nicht bearbeiten.'
+      504: 'Gateway Timeout: Der vorgelagerte Server konnte die Anfrage nicht rechtzeitig beantworten.'
+      default: Unsere Webseite macht eine kleine Pause.
```

Comment:
Why is it not "Unsere Webseite macht eine Pause. Du tust uns einen großen Gefallen, wenn Du es hier meldest" ??